bin/phpbench.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+php.exe "%~dp0\phpbench.php" %*
```

Comment:
Maybe try removing the vendor directory, removing composer.lock and
doing a `composer install`, perhaps ?

On Thu, Jan 14, 2016 at 04:51:29AM -0800, Remo Laubacher wrote:
>    In [1]bin/phpbench.bat:
> 
>  > @@ -0,0 +1,2 @@
>  > +@echo off
>  > +php.exe "%~dp0\phpbench.php" %*
> 
>    Sorry, the above is wrong. This doesn't work
>    C:\Work\phpbench>bin\phpbench.
>    Too many open projects /-:
>    It works because composer creates a bat file when installing it as a
>    library.
>    [2]https://getcomposer.org/doc/articles/vendor-binaries.md#what-about-windows-and-bat-files-
> 
>    However, when I run composer update it doesn't seem to create that file.
> 
>    —
>    Reply to this email directly or [3]view it on GitHub.
> 
>    Reverse link: [4]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/phpbench/phpbench/pull/298#discussion_r49722027
>    2. https://getcomposer.org/doc/articles/vendor-binaries.md#what-about-windows-and-bat-files-
>    3. https://github.com/phpbench/phpbench/pull/298/files#r49722027
>    4. https://github.com/phpbench/phpbench/pull/298/files#r49722027